MinecraftClient/Protocol/IMinecraftCom.cs

```diff
@@ -144,9 +144,10 @@ public interface IMinecraftCom : IDisposable, IAutoComplete
         /// </summary>
         /// <param name="windowId">Id of the window being clicked</param>
         /// <param name="slotId">Id of the clicked slot</param>
+        /// <param name="buttom">0 for left click, 1 for right click, 2 for middle click</param>
         /// <param name="item">Item in the clicked slot</param>
         /// <returns>True if packet was successfully sent</returns>
-        bool SendClickWindow(int windowId, int slotId, Item item);
+        bool SendClickWindow(int windowId, int slotId, byte buttom, Item item);
```

Comment:
I think maybe other actions would be impemented at a later time so how about renaming to `SendWindowAction` and using either an Enum _or_ raw protocol values instead of auto-changing 2 to 3? The parameter name would be `WindowAction action` or `byte action` 😉